src/main/java/frc/team2412/robot/Subsystems.java

```diff
@@ -18,14 +18,15 @@
 
 public class Subsystems implements Loggable {
     public static class SubsystemConstants {
-        public static final boolean CLIMB_ENABLED = false;
-        public static final boolean DRIVE_ENABLED = true;
-        public static final boolean DRIVER_VIS_ENABLED = false;
-        public static final boolean SHOOTER_VISION_ENABLED = false;
-        public static final boolean INDEX_ENABLED = false;
-        public static final boolean INTAKE_ENABLED = false;
-        public static final boolean SHOOTER_ENABLED = false;
-        public static final boolean SHOOTER_TESTING = false;
+        public static boolean CLIMB_ENABLED = false;
+        public static boolean DRIVE_ENABLED = true;
+        public static boolean DRIVER_VIS_ENABLED = false;
+        public static boolean SHOOTER_VISION_ENABLED = true;
+        public static boolean INDEX_ENABLED = true;
+        public static boolean INTAKE_ENABLED = true;
+        public static boolean SHOOTER_ENABLED = true;
+        public static boolean SHOOTER_TESTING = false;
```

Comment:
These are never written, so this just means there's more code compiled in and the compiler can't short-circuit the comparison statements where they are used.